crates/weaver_forge/src/lib.rs

```diff
@@ -119,8 +98,8 @@ impl Object for TemplateObject {
 /// Template engine for generating artifacts from a semantic convention
 /// registry and telemetry schema.
 pub struct TemplateEngine {
-    /// Template path
-    path: PathBuf,
+    /// File loader used by the engine.
+    file_loader: Arc<dyn FileLoader + 'static>,
```

Comment:
nit: Here can you add the `+Send + Sync`?

Reply:
See [9b8e7a7](https://github.com/open-telemetry/weaver/pull/171/commits/9b8e7a7564c059510f63b361f8336b485f5ef12c)